src/stackwalk.c

```diff
@@ -84,8 +84,28 @@ size_t rec_backtrace(uintptr_t *data, size_t maxsize)
     return rec_backtrace_ctx(data, maxsize, &context);
 }
 
+int jl_num_frames(uintptr_t ip, int skipC)
```

Comment:
I'd be open to having a copy of `firstcaller` written in C. I'm not really interested in simply truncating backtraces.